assets/release/release-aarch64.json

```diff
@@ -15,6 +15,11 @@
     "csi-livenessprobe": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:6451a940b9c21e58364aa1445c18ae94438facfb45b4141301d8c8dbaa34eeda",
     "ovn-kubernetes-microshift": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:94f1030c585e7157725064e58f162eb7e17742ecbeb949a841cccab8fa6f795b",
     "pod": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:2962383567ed9f2b78b394562c4f8c03ec511906da7f855e98d2ac418d6d6258",
-    "service-ca-operator": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:2ddea0c30e6facf46d52015482a7f0914a8ec0182cff3f0d5ec4a4f6035114e8"
+    "service-ca-operator": "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:2ddea0c30e6facf46d52015482a7f0914a8ec0182cff3f0d5ec4a4f6035114e8",
+    "topolvm-csi": "registry.redhat.io/lvms4/topolvm-rhel8@sha256:10bffded5317da9de6c45ba74f0bb10e0a08ddb2bfef23b11ac61287a37f10a1",
+    "topolvm-csi-registrar": "registry.redhat.io/openshift4/ose-csi-node-driver-registrar@sha256:a4319ff7c736ca9fe20500dc3e5862d6bb446f2428ea2eadfb5f042195f4f860",
+    "topolvm-csi-livenessprobe": "registry.redhat.io/openshift4/ose-csi-livenessprobe@sha256:9df24be671271f5ea9414bfd08e58bc2fa3dc4bc68075002f3db0fd020b58be0",
+    "topolvm-csi-resizer": "registry.redhat.io/openshift4/ose-csi-external-resizer@sha256:9d486daffd348664c00d8b80bd0da973b902f3650acdef37e1b813278ed6c107",
+    "topolvm-csi-provisioner": "registry.redhat.io/openshift4/ose-csi-external-provisioner@sha256:199eac2ba4c8390daa511b040315e415cfbcfa80aa7af978a33624445b96c17c"
```

Comment:
We currently have the same images without "topolvm-" prefix. There seems to be some duplication here. Should the old names be removed?

Reply:
The topolvm name here reflects the container names in the upstream OCP manifests and means we don't have to maintain a mapping of OCP -> uShift images.

Reply:
So, we should probably remove all the csi-* images from the JSON files?

Reply:
My mistake, I hadn't caught what you meant.  Will remove the old image tags.